instance/indicators.js

```diff
@@ -20,13 +20,19 @@ const indicators = process.env.NODE_ENV === 'production' || process.env.USE_PROD
   {
     id: 'od-education',
     name: 'Education',
-    description: 'The importance of road segments to get to educational facilities',
+    description: 'The importance of road segments to get from a populated place to the closest educational facility',
     defaultValue: 3
   },
   {
     id: 'od-healthcare',
     name: 'Healthcare',
-    description: 'The importance of road segments to get to healthcare facilities',
+    description: 'The importance of road segments to get from a populated place to the closest healthcare facility',
+    defaultValue: 3
+  },
+  {
+    id: 'od-healthcare',
```

Comment:
Duplicate id
```suggestion
    id: 'od-markets',
```

Reply:
Resolved through commit below. It should be `od-market` (singular) to match what's on S3.